### ISSUE TITLE
Add important descriptors on www.julialang.org

### ISSUE THREAD

**Issue by xorJane** [JuliaLang/www.julialang.org#135]
Under "Julia in a nutshell", we say that Julia is fast, dynamic, optionally typed, general, technical, and composable.

Two important descriptors we should add and highlight are "easy  to learn" and "open source". These sections might read

> **Easy to learn**
Julia has high level syntax, making it an accessible language for programmers from any background or experience level.

>  **Open source**
Julia is free for everyone to use, and all source code is publicly viewable on GitHub.

@ahclaster, does that wording sound reasonable?

**Comment by ararslan** [comment]
I'd reword the former a bit:
> **Easy to learn**
> Julia has high level syntax, making it an accessible language for programmers from any background or experience level.

Also, I don't find "you can avoid licensing" particularly clear.

**Comment by xorJane** [comment]
I like your reword and comment! Updated above. :)

**Comment by ViralBShah** [comment]
May need a bit of a redesign also - maybe 4x2?

**Comment by ViralBShah** [comment]
I think "Technical" and "Composable" can be replaced with "Easy to Learn" and "Open Source".
